### PR TITLE
Docs, SM64, SC2: Minor Documentation Updates

### PR DESCRIPTION
**Pull request by Bicoloursnake**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

Docs: removed an extraneous sentence from SC2 setup guide and corrected a misspelling in SM64 info doc.

## What is this fixing or adding?
Making bits of documentation easier to read

## How was this tested?
Looking at the rendered markdown.

### CODE REVIEW

**Comment by ThePhar** [comment]
SM64 doc change is fine, as it's just a typo fix. 

As for removing the SC2 line, @Ziktofel are you alright with that?

**Comment by Ziktofel** [comment]
I'm not a mac user myself, so a mac user should review it if it's OK. I don't mind as long as this stays correct

**Comment by Bicoloursnake** [comment]
I wrote the mac section of that guide after testing that it even worked in the first place. The sentence being removed was initially included to explain why the command included a 'sudo', after some further testing, I realized that the 'sudo' wasn't necessary, but forgot to remove the sentence explaining why I had the sudo before it got merged in (See #1747 ) If we can find another mac user to test it, that'd be fantastic, but there's a nonzero chance I'm the only one who has tested this on mac.

**Review by black-sliver — APPROVED** [review]

**Pull request merged**
